### PR TITLE
feat: PartyKit integration on drop page

### LINKED ISSUE #9360: PartyKit integration on drop page
Let's party a bit 
image

## Integration
https://www.partykit.io/
https://github.com/partykit/partykit
https://docs.partykit.io/

## Design
- default cursors, like from the example with flags 
image
- the cursor will have random colors - not sure if its default behavior or not? 
- instead of flag, it shows dot spent in small container - please round to the whole number
image

figma link: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=8127-21723&mode=design&t=W4cp7VefogMZZOIv-11

## Notes
- cursor will have 20% opacity over the cards section
this: 
image

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context
❗️ update `PARTY_SOCKET_HOST` inside `composables/party/useParty.ts` with a deploy url from kodadot 

- [x] Closes #9360
- [x] Needs https://github.com/kodadot/workers/pull/251

## Needs Design check

- @exezbcz please review

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-02-15 at 17 11 17@2x

### CODE REVIEW

**Comment by exezbcz** [comment]
party 
image

who is it?

**Comment by hassnian** [comment]
> party image
> 
> who is it?

it's me :d

**Comment by yangwao** [comment]
> ❗️ update `PARTY_SOCKET_HOST` inside `composables/party/useParty.ts` with a deploy url from kodadot

quick setup what I need to do to be done?

File: composables/party/useParty.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import PartySocket from 'partysocket'
+
+type UsePartyParams<T> = {
+  room: Ref<string>
+  onMessage?: (value: T) => void
+}
+
+const PARTY_SOCKET_HOST = 'https://my-party.hassnian.partykit.dev/'
```
Comment (yangwao) [inline composables/party/useParty.ts:8]:
```suggestion
const PARTY_SOCKET_HOST = 'https://partykit.kodadot.workers.dev'
```

**Comment by yangwao** [comment]
@hassnian any chance why it doesn't work here? 
- https://deploy-preview-9413--koda-canary.netlify.app/ahp/drops/echo

**Comment by hassnian** [comment]
> @hassnian any chance why it doesn't work here?
> 
> * https://deploy-preview-9413--koda-canary.netlify.app/ahp/drops/echo

checking

**Comment by yangwao** [comment]
on Sailwings it has worked

**Comment by yangwao** [comment]
I don't even see that it would opened on Echo drop socket 

image

**Comment by hassnian** [comment]
> I don't even see that it would opened on Echo drop socket
> 
> image

Ic the issue one sec

**Comment by yangwao** [comment]
Yes, I think Echo is holderOf drop so different page? Can we enable it there as well please?

**Comment by yangwao** [comment]
wondering why 0 DOT is shown? 

image

**Comment by hassnian** [comment]
> wondering why 0 DOT is shown?
> 
> image

>  it shows dot spent in small container

CleanShot 2024-02-15 at 18 46 57@2x

**Comment by yangwao** [comment]
okay let's go then 

Thanks!
pay 80 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $80 @ 7.86 USD/DOT ~ 10.178 $DOT
🧗 [16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD](https://polkadot.subscan.io/account/16faLfsywwNATaEfbH2ah75dn6ZmctQWpMS5G4KFhbmj5hnD)
🔗 [0x60eeced0a56e317873b2323417329046851ba8bb94056d84607d2027118043a2](https://polkadot.subscan.io/extrinsic/0x60eeced0a56e317873b2323417329046851ba8bb94056d84607d2027118043a2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**